content/beginner/194_secrets_manager/prepare-secret.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Prepare secret and IAM access controls"
+date: 2021-10-01T00:00:00-04:00
+weight: 20
+pre: '<i class="fa fa-film" aria-hidden="true"></i>'
+draft: false
+---
+
+#### Set variables
+Verify AWS_REGION variable. 
+```bash
+test -n "$AWS_REGION" && echo AWS_REGION is "$AWS_REGION" || echo AWS_REGION is not set
+```
+If not set, then complete [Start the workshop instructions](/020_prerequisites/workspaceiam/) and verify again.
+
+Set the cluster name variable.
+```bash
+export EKS_CLUSTERNAME="eksworkshop-eksctl"
+```
+
+#### Create a test secret with the AWS Secrets Manager.
+```bash
+aws --region "$AWS_REGION" secretsmanager \
+  create-secret --name DBSecret_eksworkshop \
+  --secret-string '{"username":"foo", "password":"super-sekret"}'
+
```

Comment:
Remove this extra blank line (it gets copied)

Reply:
Removed extra blank line from aws command line

Changes applied with https://github.com/aws-samples/eks-workshop/pull/1271/commits/e080dd759e1dd91a557d0ea770aede7a44e94405